Jenkinsfile

```diff
@@ -47,15 +47,13 @@ pipeline {
                         CHANGE_BRANCH_ENCODED = java.net.URLEncoder.encode(CHANGE_BRANCH, 'UTF-8')
                         def prDetails = readJSON(text: httpRequest(url: GITHUB_API + '/' + REPO + '/pulls?head=brave:' + CHANGE_BRANCH_ENCODED, customHeaders: GITHUB_AUTH_HEADERS, quiet: true).content)[0]
                         SKIP = prDetails.draft.equals(true) || prDetails.labels.count { label -> label.name.equalsIgnoreCase('CI/skip') }.equals(1) || prDetails.labels.count { label -> label.name.equalsIgnoreCase("CI/skip-${PLATFORM}") }.equals(1)
-                        RUN_NETWORK_AUDIT = prDetails.labels.count { label -> label.name.equalsIgnoreCase('CI/run-network-audit') }.equals(1)
```

Comment:
@mihaiplesa Based on our discussion on Slack, with further clarification from @bridiver, I think this change it's not needed. Looks like what needs to be changes should happen in Jenkins itself, which it's what I think https://github.com/brave/devops/issues/5965 is for.

Removing this and force-pushing